node-memory-considerate-loading/public/app.css

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.App {
+  text-align: center;
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
```

Comment:
Because we rely on WebPageTest / real device testing to test this demo out, I would like to make sure the layout is a little more optimized for smaller viewports. In the below, you'll notice that App-header's `min-height` is pushing a lot of the content down to the center:

![image](https://user-images.githubusercontent.com/110953/65064668-c306af00-d935-11e9-896b-f2d7036ab5b6.png)

Could we change this to 50 and also change the body background color to match `#282c34`?

![image](https://user-images.githubusercontent.com/110953/65064770-f21d2080-d935-11e9-8d9c-55fed3594198.png)

Reply:
Yes I updated the CSS.